src/static/helpers/slasClient.ts

```diff
@@ -109,3 +109,19 @@ export interface ISlasClient {
 
   clientConfig: ClientConfig;
 }
+
+/**
+ * Custom query parameter type with any string prefixed with `c_` as the key and the allowed
+ * types for query parameters for the value.
+ */
+export type CustomQueryParameters = {
+  [key in `c_${string}`]: string | number | boolean | string[] | number[];
```

Comment:
curious, what's the use case for `string[]` and `number[]`?

Reply:
We're allowing the same query parameter values as defined in the rest of the SDK: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/da2f6ca1828dafe6625a15751b2c7aedfbf532c9/src/static/helpers/types.ts#L54

`string[]` and `number[]` are valid types for query parameters and we'll want to allow customers to pass those into their API call if they've configured their environment to handle those custom parameters